.ci/travis/checkchmod.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+CHMOD=$(find -type f -not -path '*/\.*' -a -type f -not -name '*.sh' -a \( -type d -not -perm 775 -o -type f -not -perm 664 \))
```

Comment:
please make pattern as `'*/\.git/*'` , to skip only git files and keep under controll all other files, we have few folders that starts with "." already.